src/scripts-upload-media/fearspeech.js

```diff
@@ -0,0 +1,59 @@
+const axios = require("axios");
+const path = require("path");
+const { v4: uuid } = require("uuid");
+const fs = require("fs")
+require("dotenv").config({
+    path: path.join(__dirname, "development.env"),
+});
+const stories = require("./fear_speech_data.json")
+
+const accessToken = process.env.ACCESS_TOKEN;
+const datasource = "efb05069-e220-4b82-8849-98d14bffdafd";
+
+const failedRequestsFile = "fear-speech-failed-requests.json";
+const insertPost = async () => {
+    const posts = []
+    for (const post of Object.values(stories)) {
+        const fileId = uuid();
+        const { uploadData } = require("./s3");
+        const awsResponse = await uploadData(post.message_text, fileId);
+        post.s3URL = awsResponse.Location
```

Comment:
same feedback. Replace this with post.s3uRL = https://fs.tattle.co.in/service/kosh/file/${fileId}